src/pages/ViewScholarship.jsx

```diff
@@ -47,6 +49,30 @@ export default function ViewScholarship({ history, location, match }) {
       <Helmet>
         <title>{scholarship.data.name}</title>
       </Helmet>
+
+      {location?.state?.status && (
+        <Alert
+          color="primary"
+          variant="filled"
+          action={
+            <>
+              <IconButton
+                size="medium"
+                color="inherit"
+                onClick={() =>
+                  history.replace(`/scholarships/${scholarship.id}`)
+                }>
+                <CloseIcon fontSize="small" />
+              </IconButton>
+            </>
+          }>
+          <AlertTitle sx={{ color: '#fff' }}>Success</AlertTitle>
+          {`Scholarship successfully ${
+            location?.state?.status === 'add' ? 'submitted.' : 'updated.'
+          }`}
+        </Alert>
+      )}
```

Comment:
<img width="1188" alt="Screen Shot 2021-12-19 at 7 37 37 PM" src="https://user-images.githubusercontent.com/40483569/146708243-d3064ac6-82e3-4b5a-a735-83fb4097f3f1.png">

This is what is currently looks like. I didn't add the action buttons since they are easy to achieve in the navbar.